persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/impl/interceptors/DefaultQueryInterceptor.java

```diff
@@ -252,9 +253,12 @@ private void applyState(
       case RPC_READY:
         notifyRpcChange(endpoint, epState.isRpcReady());
         break;
+      default:
+        // ignore other states
```

Comment:
Can someone confirm that this is the intended behavior?

Reply:
Yes, pretty sure that's ok (Gossip has quite a few states, not all of which we care for what the interceptor does).